README.md

```diff
@@ -1,40 +1,79 @@
 
-## Core Projects
+* **[The LAOS Parachain](#the-laos-parachain-monorepo)**. You are already in it. To streamline the development process and encourage diverse contributions, all core components of the Parachain were integrated in this single monorepo.
 
-### [Ownership Chain](./ownership-chain/)
+* **[The Universal Node](https://github.com/freeverseio/laos-universal-node)**. This node streamlines the integration process for DApps aiming to incorporate bridgeless minting and evolution across various chains, including Ethereum, by merely adjusting the RPC endpoint to connect to the relevant Universal Nodes.
 
-Ownership Chain is dedicated to the enforcement and management of asset ownership within the LAOS ecosystem. It ensures secure, transparent, and efficient tracking of asset ownership changes, serving as a cornerstone of the LAOS platform.
+* **[The Universal ERC721 Solidity Templates](https://github.com/freeverseio/laos-erc721)**. This template contains the minimal extensions of the ERC721 OpenZeppelin implementation that enables bridgeless minting and evolution in any EVM chain.
 
-### [Evolution Chain](./evolution-chain/)
+LAOS is targeting late Q1'2024 to bid for a Parachain slot on Polkadot.
+Presently, LAOS is accessible through its test network, named **Caladan**. Additionally, it is in the process of participating in a Parachain auction on Kusama; the Parachain will be named **KLAOS**, which stands for **K**usama LAOS. 
 
-Evolution Chain is engineered to provide dynamic capabilities to NFTs. It enables assets to evolve and adapt over time, based on either pre-defined rules or real-time external factors. This adds a life-like quality to NFTs, making them more engaging and valuable.
+### Caladan Parachain: 
+* Substrate RPC endpoint: [wss://caladan.gorengine.com/own](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fcaladan.gorengine.com%2Fown#/explorer)
+* EVM endpoint: https://caladan.gorengine.com/own
+* EVM Chain ID: 667
+* EVM Currency Symbol: CLD 
+* EVM block explorer: https://caladan.gorengine.com 
 
-### [Bridge](./bridge/)
+### KLAOS Parachain: 
+* Substrate RPC endpoint: TBD
+* EVM endpoint: TBD
+* EVM Chain ID: TBD
+* EVM Currency Symbol: KLAOS 
+* EVM block explorer: TBD
 
-The Bridge module acts as the communication layer between the Ownership Chain and Evolution Chain, allowing for seamless data exchange and functionality integration.
+## Running your own node
 
-## Testing Networks
+The quickest entry point to run your own LAOS Parachain node:
+```
+$ docker run freeverseio/laos-ownership-node:<release> --chain=caladan
+```
+The available releases are published [here](https://github.com/freeverseio/laos/releases).
 
-#### Caladan Ownership chain: 
-* Polkadot.js explorer: [wss://caladan.gorengine.com/own](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fcaladan.gorengine.com%2Fown#/explorer)
-* EVM endpoint: https://caladan.gorengine.com/own
-* EVM blockexplorer: https://caladan.gorengine.com 
+# Contributing
```

Comment:
I don't think this part is needed and if it's needed is better to point to an official guide : https://docs.github.com/en/get-started/quickstart/contributing-to-projects

Reply:
change applied